pages/index.js

```diff
@@ -238,11 +219,10 @@ export default function Home() {
       <div className={walletAddress ? 'authed-container' : 'container'}>
         <div className="header-container">
           <p className="header">🖼 GIF Portal</p>
-          <p className="sub-text">
-            {!walletAddress ? "Upload" : "View"} your GIF collection in the metaverse ✨
+          <div className="sub-text">
```

Comment:
userStrip returns div which shouldn't be nested in <p>, converting it to div solved that